src/library-authoring/backup-restore/LibraryBackupPage.test.tsx

```diff
@@ -0,0 +1,245 @@
+import {
+  act,
+  initializeMocks,
+  render,
+  screen,
+} from '@src/testUtils';
+import userEvent from '@testing-library/user-event';
+import { LibraryBackupStatus } from './data/constants';
+import { LibraryBackupPage } from './LibraryBackupPage';
+import messages from './messages';
+
+// Mock the hooks/context used by the page so we can render it in isolation.
+jest.mock('@src/library-authoring/common/context/LibraryContext', () => ({
+  useLibraryContext: () => ({ libraryId: 'lib:TestOrg:test-lib' }),
+}));
```

Comment:
It's OK to mock this but the mocked data here is missing most of the values, and may end up causing errors in the tests if it ever tries to access any data besides the `libraryId`. I would normally recommend just providing the library context to the test, as we do in other tests:

```tsx
import { render as baseRender } from '../../testUtils';

import { mockContentLibrary } from '../data/api.mocks';
mockContentLibrary.applyMock();

const render = (libraryId: string = mockLibraryId) => baseRender(<CollectionInfoHeader />, {
  extraWrapper: ({ children }) => (
    <LibraryProvider libraryId={mockContentLibrary.libraryId}>{ children }</LibraryProvider>
  ),
});
```

Then you can just use `render()` in the test, and it will have correctly-typed, realistic data.

Reply:
Also note that `mockContentLibrary` provides many different library IDs useful for tests, such as libraries with error states or read-only or unpublished changes, etc.: `mockContentLibrary.libraryIdReadOnly`, `mockContentLibrary.libraryIdThatNeverLoads`, `mockContentLibrary.library404`, `mockContentLibrary.libraryUnpublishedChanges`, etc.